solidus_braintree.gemspec

```diff
@@ -19,8 +19,8 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "solidus_api", [">= 1.0.0", "< 3"]
-  spec.add_dependency "solidus_core", [">= 1.0.0", "< 3"]
+  spec.add_dependency "solidus_api", [">= 1.0.0", "<= 3.4"]
```

Comment:
WDYT about using the same syntax and just changing the number to 4?
```
spec.add_dependency "solidus_api", [">= 1.0.0", "< 4"]
```
Because in some bundler versions, it could be resolved only to 3.4.0 and not to the latest 3.4.x version

Reply:
This makes sense

https://github.com/solidusio/solidus_braintree/pull/147/commits/9e297777d9fbfe56bccbc48a45515f3233a79603